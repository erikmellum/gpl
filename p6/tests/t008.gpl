string window_title = "p6 t008.gpl -- compare to t008.jpg";

circle c00(x = floor(100 * cos(00)) + 200, y = floor(100 * sin(00)));
circle c01(x = floor(100 * cos(20)) + 200, y = floor(100 * sin(20)));
circle c02(x = floor(100 * cos(40)) + 200, y = floor(100 * sin(40)));
circle c03(x = floor(100 * cos(60)) + 200, y = floor(100 * sin(60)));
circle c04(x = floor(100 * cos(80)) + 200, y = floor(100 * sin(80)));
circle c05(x = floor(100 * cos(100)) + 200, y = floor(100 * sin(100)));
circle c06(x = floor(100 * cos(120)) + 200, y = floor(100 * sin(120)));
circle c07(x = floor(100 * cos(140)) + 200, y = floor(100 * sin(140)));
circle c08(x = floor(100 * cos(160)) + 200, y = floor(100 * sin(160)));
circle c10(x = floor(100 * cos(180)) + 200, y = floor(100 * sin(180)));
