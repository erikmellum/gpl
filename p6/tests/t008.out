gpl.cpp::main()
  input file(tests/t008.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(true, file = results/t008.mypixels)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
game_object c00
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(300),
    y = int(0)
  }
game_object c01
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(293),
    y = int(34)
  }
game_object c02
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(276),
    y = int(64)
  }
game_object c03
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(250),
    y = int(86)
  }
game_object c04
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(217),
    y = int(98)
  }
game_object c05
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(182),
    y = int(98)
  }
game_object c06
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(150),
    y = int(86)
  }
game_object c07
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(123),
    y = int(64)
  }
game_object c08
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(106),
    y = int(34)
  }
game_object c10
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(100),
    y = int(0)
  }
string window_title "p6 t008.gpl -- compare to t008.jpg"
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().

Printing the symbol table from signal_handler()
game_object c00
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(300),
    y = int(0)
  }
game_object c01
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(293),
    y = int(34)
  }
game_object c02
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(276),
    y = int(64)
  }
game_object c03
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(250),
    y = int(86)
  }
game_object c04
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(217),
    y = int(98)
  }
game_object c05
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(182),
    y = int(98)
  }
game_object c06
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(150),
    y = int(86)
  }
game_object c07
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(123),
    y = int(64)
  }
game_object c08
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(106),
    y = int(34)
  }
game_object c10
  Circle
  {
    animation_block = animation_block(0),
    blue = double(1),
    drawing_order = int(0),
    green = double(0),
    h = int(20),
    proximity = int(4),
    radius = int(10),
    red = double(0),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(20),
    x = int(100),
    y = int(0)
  }
string window_title "p6 t008.gpl -- compare to t008.jpg"
